### DISCUSSION TITLE
Reset mouse to center of screen (and other mouse specific locations)

### DISCUSSION THREAD

**Discussion by GaMatecal** [AntiMicroX/antimicrox#960] [Q&A]
Is it possible to reset mouse to center of screen?
Use case. While moving the joystick, the mouse is moving.  But when I stop using the joystick, I'd like it to reset to center of screen.

Additionally, is it possible to assign buttons to click a specific spot on the screen?

I've been using AutoHotKey, but the mouse control on it is a bit wonky and jittery.  AntiMicroX has very smooth mouse control, and looking to implement what I've done with AutoHotKey to AntiMacroX

**Comment by pktiuk** [comment]
You can set mouse as a spring button.

But currently spring center is mapped to the center of the screen
https://github.com/AntiMicroX/antimicrox/issues/648
